src/Cart/Checkout/AddressManagement/AddAddressModel/AddAddressModel.js

```diff
@@ -0,0 +1,116 @@
+import "./AddAddressModel.css";
+import { useState } from "react";
+import { Button } from "forkui-lib";
+import { IndianStates } from "./StateDatabases/IndianStates";
+import { USStates } from "./StateDatabases/USStates";
+import { AustralianStates } from "./StateDatabases/AustralianStates";
+import { Link } from "react-router-dom";
+export const AddAddressModel = ({
+  setShowAddAddressModel,
+  addresses,
+  setAddresses
+}) => {
+  const [country, setCountry] = useState("India");
+  const [state, setState] = useState("");
+  const [name, setName] = useState("");
+  const [address, setAddress] = useState("");
+  const [city, setCity] = useState("");
+  const [zip, setZip] = useState(0);
+  const [mob, setMob] = useState(0);
+  const updateState = (stateSelected) => {
+    setState(stateSelected);
+  };
+  const submitNewAddress = () => {
+    console.log([
+      ...addresses,
+      {
+        id: addresses.length + 1,
+        country,
+        state,
+        name,
+        address,
+        city,
+        zip,
+        mob
+      }
```

Comment:
Instead of using useState for all the address fields ( country,state,name etc) you can define an object once in one useState that contains all these fields & update the values of these fields when new address is added

Reply:
Yup! Done with this refractor